plugins/modules/volume.py

```diff
@@ -145,7 +147,7 @@
   returned: state == present
   type: dict
   sample: {'slug': 'lpg1'}
-server_uuids:
+servers:
```

Comment:
hmm server_uuids is still returned as list of keys, but servers is also returned as objects from the api or did I miss a thing?

Reply:
You are right. Will fix this. Thanks for spotting this.